### PR TITLE
Try running capybara tests against dockerized instance

### LINKED ISSUE #1858: docker gollumwiki/gollum:master crashes
Since the last commit on 13.09.22 to [gollumwiki/gollum:master]([master](https://registry.hub.docker.com/layers/gollumwiki/gollum/master/images/sha256-fb426b78848e8f01f9d3a9e6b230b7a48f824740329985d3eb50508fa3e35b90?context=explore)) the docker container crashes

```bash
alexander@ubuntu:~$ docker start -a gollum 
WARNING: Mathjax will soon be disabled by default. To explicitly enable it, use --mathjax
/usr/local/bundle/gems/execjs-2.8.1/lib/execjs/runtimes.rb:58:in `autodetect': Could not find a JavaScript runtime. See https://github.com/rails/execjs for a list of available runtimes. (ExecJS::RuntimeUnavailable)
	from /usr/local/bundle/gems/execjs-2.8.1/lib/execjs.rb:5:in `<module:ExecJS>'
	from /usr/local/bundle/gems/execjs-2.8.1/lib/execjs.rb:4:in `<top (required)>'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/bundle/gems/uglifier-4.2.0/lib/uglifier.rb:5:in `<top (required)>'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/bundle/gems/gollum-5.3.0/lib/gollum/assets.rb:2:in `<top (required)>'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/bundle/gems/gollum-5.3.0/lib/gollum/app.rb:16:in `<top (required)>'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	from /usr/local/bundle/gems/gollum-5.3.0/bin/gollum:279:in `<top (required)>'
	from /usr/local/bundle/bin/gollum:23:in `load'
	from /usr/local/bundle/bin/gollum:23:in `<main>'
```

Please let me know if I need to report the bug somewhere else.

BR
Alex

### PR DESCRIPTION
**Pull request by dometto**
Resolves #1858 #1853

It looks like the tests we are running in the docker test workflow (running `gollum --irb`) isn't  robust enough to pick up failures. This PR attempts to instead run the capybara tests against an instance of gollum running on the created docker image.

### CODE REVIEW

**Comment by dometto** [comment]
@benjaminwil see [ac6f37c](https://github.com/gollum/gollum/pull/1860/commits/ac6f37ca24ac72b7bf3bfef812a46b9b03049fee) for the fix to #1853. This took me a while to figure out because it was hard to get Actions to do what I want, and on the way I also ran into two other problems, fixed in [f161e03](https://github.com/gollum/gollum/pull/1860/commits/f161e033974cbe63fb0366e42b71f927d0c1f38f) and https://github.com/gollum/gollum/pull/1860/commits/49c491a85a71c33f0a15055c58c988255f978d66.

My general approach here is to run gollum from the container built in this task, then run the integration tasks against the Dockerized instance.

File: lib/gollum.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ module Gollum
   KEYBINDINGS = ['default', 'vim', 'emacs']
   
   ::I18n.available_locales = [:en]
-  ::I18n.load_path = Dir[::File.expand_path("lib/gollum/locales") + "/*.yml"]
+  ::I18n.load_path = Dir[::File.expand_path("../gollum/locales",  __FILE__) + "/*.yml"]
```
Comment (benjaminwil) [inline lib/gollum.rb:20]: 🥲 I'm sorry.

Comment (dometto) [reply lib/gollum.rb:20]: No worries! Also slipped through my review :) And once I had entangled this from the other issues, it was actually quickly discovered.

**Comment by dometto** [comment]
Just fur future information, the docker-test workflow now:

* builds the docker image
* runs `bundle install` locally (i.e. on the checked out gollum repo) to get `capybara` and any other gems the integration tests might need
* runs gollum from the built container, on `lotr.git` (which is first cloned to `/tmp`)
* runs `bundle exec rake test:capybara` while setting the new `GOLLUM_CAPYBARA_URL` to `http://locahost:4567`, which is where gollum is running inside the Docker container

There may be more efficient ways of running capybara, but this works for now. The setup does assume that all integration tests are non-destructive, i.e. make no changes to the repository that we're testing with. If this ever changes, the integration tests in this workflow will likely fail. We can then come up with a more complicated solution.

**Pull request merged**